api/src/main/java/org/pmiops/workbench/db/model/DbDemographicSurvey.java

```diff
@@ -174,6 +176,34 @@ public void setRaceEnum(List<Race> raceList) {
         raceList.stream().map(DemographicSurveyEnum::raceToStorage).collect(Collectors.toList());
   }
 
+  @ElementCollection(fetch = FetchType.LAZY)
+  @CollectionTable(
+      name = "demographic_survey_gender",
+      joinColumns = @JoinColumn(name = "demographic_survey_id"))
+  @Column(name = "gender")
+  public List<Short> getGender() {
+    return gender;
+  }
+
+  public void setGender(List<Short> gender) {
+    this.gender = gender;
+  }
+
+  @Transient
+  public List<Gender> getGenderEnum() {
+    if (gender == null) return null;
```

Comment:
Please don't return a null list if you can avoid it. Can you use `Collections.emptyList()`?